packages/nuxt/src/runtime/plugins/database.server.ts

```diff
@@ -0,0 +1,205 @@
+import {
+  type Span,
+  type StartSpanOptions,
+  addBreadcrumb,
+  captureException,
+  debug,
+  flushIfServerless,
+  SEMANTIC_ATTRIBUTE_SENTRY_OP,
+  SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN,
+  SPAN_STATUS_ERROR,
+  startSpan,
+} from '@sentry/core';
+import type { Database, PreparedStatement } from 'db0';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { defineNitroPlugin, useDatabase } from 'nitropack/runtime';
+// @ts-expect-error - This is a virtual module
+import { databaseInstances } from '#sentry/database-config.mjs';
+
+/**
+ * Keeps track of prepared statements that have been patched.
+ */
+const patchedStatement = new WeakSet<PreparedStatement>();
+
+/**
+ * The Sentry origin for the database plugin.
+ */
+const SENTRY_ORIGIN = 'auto.db.nuxt';
+
+/**
+ * Creates a Nitro plugin that instruments the database calls.
+ */
+export default defineNitroPlugin(() => {
+  try {
+    debug.log('@sentry/nuxt: Instrumenting databases...');
```

Comment:
Please add conditionals to the `debug.log` statements so they are only shown in debug mode. And I think the log already includes `[Sentry]` at the start (but not 100%) so we could just simplify it so it shows something like this in the end: `[Sentry] [Nitro Database Plugin]: Instrumenting databases...`

Reply:
Forget the note about conditionally calling it. `debug` already handles that

https://github.com/getsentry/sentry-javascript/pull/17858/files#r2418913504